web_widget_x2many_2d_matrix/static/src/js/abstract_view_matrix_limit_extend.js

```diff
@@ -0,0 +1,19 @@
+odoo.define(
```

Comment:
Actually [there's an exception on the line width for the `odoo.define` lines in the linter](https://github.com/OCA/pylint-odoo/blob/eb11004b3b6a99cca23f9c096bea284e9f2d7d02/pylint_odoo/examples/.jslintrc#L200), just for the case you have at hand.

The best you can do here is to put just a single line:

```js
 odoo.define( "web_widget_x2many_2d_matrix.matrix_limit_extend", function (require) {
```

Thanks to that exception, the linter should be OK.